playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.tsx

```diff
@@ -53,16 +50,16 @@ const RichTextEditor = (props: RichTextEditorProps) => {
   } = props
 
   const ariaProps = buildAriaProps(aria),
-  dataProps = buildDataProps(data),
-  [editor, setEditor] = useState()
+    dataProps = buildDataProps(data),
+    [editor, setEditor] = useState<any>()
 
-  const handleOnEditorReady = (editorInstance) => setEditor(editorInstance),
-  element = editor?.element
+  const handleOnEditorReady = (editorInstance: any) => setEditor(editorInstance),
```

Comment:
The library is not exporting a type for this one, Nida. The other option is to create an interface and map every property being used. I was afraid to do that and break something. That's why after that I tried to find the type from the library I used the "any". Do you think the interface (just mapping the properties we are using right now) is a better solution?

Reply:
I would try doing an interface, see if we can figure out a better option than 'any' just because we do try to avoid this type. Timebox it for 20-30 mins and reach out if it becomes a blocker